Frontend/FindGym/PlaceDetailsCall.js

```
@@ -0,0 +1,12 @@
+import axios from "../../../api/axios";
+
+ export const getPlaceDetails = async (place_id) => {
+    try {
+      const result = await axios.get("/placeDetails?place_id=" + place_id);
+      return result.data.result;
+
+    } catch (error) {
+
```

avoid empty catch blocks. this is hard to debug later.